frontend/src/pages/DetailedReviewPage/index.tsx

```diff
@@ -1,12 +1,12 @@
-import { ErrorSuspenseContainer, LoginRedirectModal } from '@/components';
+import { ErrorSuspenseContainer } from '@/components';
 import { useGroupAccessCode } from '@/hooks';
 
 import { DetailedReviewPageContents } from './components';
 
 const DetailedReviewPage = () => {
   const { groupAccessCode } = useGroupAccessCode();
 
-  if (!groupAccessCode) return <LoginRedirectModal />;
+  if (!groupAccessCode) return;
```

Comment:
`undefined`를 반환하지 않고, `null`을 명시적으로 반환하도록 하는 것이 좋을 것 같아요.
```suggestion
  if (!groupAccessCode) return null;
```

Reply:
해당 부분은 목록 페이지가 무한 스크롤을 반영 분이 pr에 올라가 있어서, 해당 pr 머지 후 pull 받을 거라 임시로 고쳤어요